client/config/constants.ts

```diff
@@ -1,6 +1,6 @@
 // APP
-export const API_URL = process.env.NODE_ENV === 'production' ? 'https://server.feednext.io/api' : 'http://localhost/api'
-export const SOCKET_URL = process.env.NODE_ENV === 'production' ? 'https://server.feednext.io' : 'http://localhost'
+export const API_URL = process.env.NODE_ENV === 'production' ? 'https://server.feednext.io/api' : 'http://localhost:3000/api'
```

Comment:
please use env variable to manage parameters, don't put magic number in code directly.